tests/backend/test_nongaussian_gates.py

```diff
@@ -18,13 +18,49 @@
 
 import numpy as np
 
-KAPPAS = np.linspace(0, 2 * np.pi, 7)
+from scipy.linalg import expm
 
+KAPPAS = np.linspace(0, 2 * np.pi, 7)
+GAMMAS = np.linspace(0, 2 * np.pi, 7)
 
 @pytest.mark.backends("fock", "tf")
 class TestFockRepresentation:
     """Tests that make use of the Fock basis representation."""
 
+    @pytest.mark.parametrize("gamma", GAMMAS)
+    def test_cubic_phase(self, setup_backend, gamma, cutoff, tol):
+        """Tests if the Cubic phase gate has the right effect on states in the Fock basis"""
+
+        backend = setup_backend(1)
+
+        backend.prepare_ket_state(np.ones([cutoff]) / np.sqrt(cutoff), 0)
```

Comment:
Nice! I've noticed that the ket prepared for the Kerr interaction test instead is normalised by `cutoff` rather than `sqrt(cutoff)`, is this a bug in `test_kerr_interaction`?

Reply:
No, `cutoff` works fine in `test_kerr_interaction` because you also use `cutoff` instead of `sqrt(cutoff)` in the reference state `ref_state = np.exp(1j * kappa * np.arange(cutoff) ** 2) / cutoff`, so everything is consistent. 

On the other hand in `test_cubic_phase` I decided to start with the correctly normalized reference state `ket1 /= np.linalg.norm(ket1)`, so I needed to use `sqrt(cutoff)`.